src/lib/components/prompt/PromptSuggestions.svelte

```diff
@@ -49,25 +56,26 @@
 		</button>
 	</div>
 	<div class="flex flex-col gap-4 overflow-auto">
-		{#await suggestionsRequest}
-			<Spinner />
-		{:then suggestions}
-			{#if suggestions === undefined || suggestions.length === 0}
-				No suggestions
-			{:else}
-				{#each suggestions as suggestion (suggestion.id)}
-					<div
-						onmouseover={() => setHoveredSuggestion(suggestion)}
-						onmouseleave={() => setHoveredSuggestion(null)}
-						onfocus={() => setHoveredSuggestion(suggestion)}
-						onblur={() => setHoveredSuggestion(null)}
-						role="button"
-						tabindex="0"
-					>
-						<PromptSuggestion {prompt} {suggestion} {editPrompt} />
-					</div>
-				{/each}
+		{#if gettingSuggestions}{:else if suggestions === undefined || suggestions.length === 0}
+			No suggestions
+		{:else if suggestionApplied > -1}
```

Comment:
I'm not sure this is the best behavior? Maybe we gray them out so you see you can't apply them? It's confusing when they all disappear I know personally I have to save the current prompt or undo it to see other suggestions but not super clear. We can grey out that apply buttons and add tooltips?